src/memory/MegaFlashRomSCCPlus.cc

```diff
@@ -270,14 +277,21 @@ unsigned MegaFlashRomSCCPlus::getFlashAddr(unsigned addr) const
 	return ((0x40000 * subslot) + tmp) & 0xFFFFF; // wrap at 1MB
 }
 
+bool MegaFlashRomSCCPlus::isSCCAccessEnabled() const
+{
+	// In both Konami and Konami SCC mappers the SCC is available (see note
+	// above)
+	return (configReg & 0xE0) == one_of(0x00, 0x20);
```

Comment:
Didn't you have `(configReg & 0x80) == 0x00` at some point? That seems closer to what Manuel Pazos mentioned in his email? But no need to change this. We can wait till we get the VHDL to really fix all details. (And not testing bit 6 is also a bit weird).

Note: the current code `(configReg & 0xE0) == one_of(0x00, 0x20)` can be simplified to `(configReg & 0xC0) == 0x00` (test that bit 6 and 7 are both 0). But totally fine to write it like this (with `one_of`). I verified that the compiler optimizes both versions equally well (it optimizes both to: `configReg <= 63`).